src/azure-cli/azure/cli/command_modules/cognitiveservices/custom.py

```diff
@@ -126,14 +135,14 @@ def create(
     if encryption is not None:
         params.properties.encryption = json.loads(encryption)
 
-    return client.create(resource_group_name, account_name, params)
+    return client.begin_create(resource_group_name, account_name, params).result()
```

Comment:
For long running operation, you don't need to add .result() if the result is not used in custom function.